lib/Doctrine/Export/Mysql.php

```diff
@@ -33,6 +33,12 @@
  */
 class Doctrine_Export_Mysql extends Doctrine_Export
 {
+    /**
+     * @var Doctrine_Connection_Mysql $conn       Doctrine_Connection object, every connection
+     *                                            module holds an instance of Doctrine_Connection
+     */
+    protected $conn;
```

Comment:
I did this to expose some methods that are specific to the Mysql connection class that this class uses.  It doesn't differ from the parent definition at all (except for the docblock).